includes/class-sensei-modules.php

```diff
@@ -266,6 +266,50 @@ private function render_module_select_for_course( $lesson_course, $lesson_module
 		return $html;
 	}
 
+	/**
+	 * Delete a term if it is childless and not associated with a lesson or course.
+	 *
+	 * @param int $module_term_id Term ID for the module.
+	 */
+	public function remove_if_unused( $module_term_id ) {
```

Comment:
would you not want to have it in a unused category rather than a full delete? just a suggestion

Reply:
Would you mind saying more? I'm not sure I follow what you mean.

Reply:
@jom don't worry it was a stupid suggestion.